### ISSUE TITLE
View element subsets, with table view and ETL🥉

### ISSUE THREAD

**Issue by pablo-mayrgundter** [bldrs-ai/Share#56]
https://docs.google.com/document/d/1JgLDfkghG6wqyYIplCQisSqyUK2XnRu43UEUEwAe8YI/edit#heading=h.rh99cx6i6qu7

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter what do you think about this issue? seems like it is lacking a description and just points to the first high-level requirements document.

**Comment by pablo-mayrgundter** [comment]
Yeah, this is getting a little long in the tooth.  This will be significantly advanced by work on CSV export, so maybe leave open and hope we can revisit soon.
